packages/ajax/src/AjaxClient.js

```diff
@@ -123,11 +115,11 @@ export class AjaxClient {
       // In this instance we actually do want to await for each sequence
       // eslint-disable-next-line no-await-in-loop
       const interceptedRequestOrResponse = await intercept(interceptedRequest);
-      if (interceptedRequestOrResponse instanceof Request) {
-        interceptedRequest = interceptedRequestOrResponse;
-      } else {
+      if (interceptedRequestOrResponse instanceof Response) {
```

Comment:
Lars refactored this code a bit in his PR that I just merged, so please rebase and check whether your change is still required.

Reply:
Cool, I will rebase